src/webview/wkwebview/mod.rs

```diff
@@ -621,6 +621,8 @@ impl Drop for InnerWebView {
         }
       }
 
+      self.navigate("about:blank");
```

Comment:
I couldn't find any other method exposed by wkwebview either.
This one looks good to me.
Could you add some comments here? So we could understand why it has to navigate to blank page before dropping.

Reply:
You are right. I added comment.
Thank you for reviewing.